src/mono/mono/mini/simd-intrinsics.c

```diff
@@ -4796,54 +4796,51 @@ emit_wasm_supported_intrinsics (
 		    id == SN_Splat && !MONO_TYPE_IS_VECTOR_PRIMITIVE(fsig->params [0]))
 			return NULL;
 
+		uint16_t op = info->default_op;
+		uint16_t c0 = info->default_instc0;
+
 		switch (id) {
 			case SN_Add:
 			case SN_Subtract:
 			case SN_Multiply:
 				return emit_simd_ins_for_binary_op (cfg, klass, fsig, args, arg0_type, id);
 			case SN_Negate:
 				return emit_simd_ins_for_unary_op (cfg, klass, fsig, args, arg0_type, id);
-			case SN_And:
-				return emit_simd_ins_for_sig (cfg, klass, OP_XBINOP_FORCEINT, XBINOP_FORCEINT_AND, arg0_type, fsig, args);
-			case SN_Bitmask:
-				return emit_simd_ins_for_sig (cfg, klass, OP_WASM_SIMD_BITMASK, -1, -1, fsig, args);
 			case SN_CompareEqual:
 				return emit_simd_ins_for_sig (cfg, klass, type_enum_is_float (arg0_type) ? OP_XCOMPARE_FP : OP_XCOMPARE, CMP_EQ, arg0_type, fsig, args);
 			case SN_CompareNotEqual:
 				return emit_simd_ins_for_sig (cfg, klass, type_enum_is_float (arg0_type) ? OP_XCOMPARE_FP : OP_XCOMPARE, CMP_NE, arg0_type, fsig, args);
 			case SN_ConvertNarrowingSignedSaturate: {
-				int intrins = -1;
 				switch (arg0_type) {
 				case MONO_TYPE_I2:
-						intrins = INTRINS_WASM_NARROW_SIGNED_V16;
+						c0 = INTRINS_WASM_NARROW_SIGNED_V16;
 						break;
 				case MONO_TYPE_I4:
-						intrins = INTRINS_WASM_NARROW_SIGNED_V8;
+						c0 = INTRINS_WASM_NARROW_SIGNED_V8;
 						break;
 				}
-				if (intrins != -1)
-						return emit_simd_ins_for_sig (cfg, klass, OP_XOP_X_X_X, intrins, arg0_type, fsig, args);
+				if (c0 != 0)
+						break;
```

Comment:
Might also be good to have comments indicating that these breaks are explicitly intended to use the fallback, like a fall-through